docs/pages/enterprise/workflow/ssh-approval-pagerduty.mdx

```diff
@@ -122,6 +139,25 @@ $ teleport-pagerduty start -d
 # DEBU   PagerDuty API health check finished ok pagerduty/main.go:176
 # DEBU   Setting up the webhook extensions pagerduty/main.go:178
 ```
+</TabItem>
+<TabItem label="Docker">
+```code
+$ docker run  -p 8081:8081  \
+-v ~/config:/var/lib/teleport/plugins/pagerduty \
+-v ~/teleport-pagerduty.toml:/etc/teleport-pagerduty.toml \
```

Comment:
These local paths will be only correct if the user ran above commands in their home directory. Maybe use absolute paths like `/tmp/config`, `/tmp/teleport-pagerduty.toml` here and above?